internal/util/config.go

```diff
@@ -0,0 +1,90 @@
+package util
```

Comment:
I'd recommend not having `util` packages, it ends up being a dumping ground for all sorts of things. Have a look at Flasher https://github.com/metal-toolbox/flasher/tree/main/internal/app or ConditionOrc https://github.com/metal-toolbox/conditionorc/tree/main/internal/app . It'd be ideal if all our apps follow a common design/layout.